static/app/components/events/interfaces/crashContent/exception/content.tsx

```diff
@@ -35,15 +40,30 @@ export function Content({
   groupingCurrentLevel,
   hasHierarchicalGrouping,
   platform,
+  projectSlug,
   values,
   type,
   meta,
 }: Props) {
+  // Organization context may be unavailable for the shared event view, so we
+  // avoid using the `useOrganization` hook here and directly useContext
+  // instead.
+  const organization = useContext(OrganizationContext);
   if (!values) {
     return null;
   }
 
+  const orgHasSourcemapCta = organization?.features?.includes('fix-source-map-cta');
+  const debugFrames = orgHasSourcemapCta
+    ? getUnqiueFilesFromExcption(values, platform, {
```

Comment:
@scttcper hmmm...I'm wondering if we should be making this API call if the FE detects source maps are set up properly

Reply:
let's just leave it as-is for now